modules/enabling-anonymous-pulls.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-anonymous-pulls_{context}"]
+= Enabling anonymous pulls
+
+The following procedure shows you how to check if your cluster has the proper roles and permissions for anonymous pulls and, if it does not, how to enable anonymous requests to the self access reviewer source.
```

Comment:
This sentence mixes up the enabling of anonymous pulls with anonymous requests to the self access reviewer resource - we should be consistent.
```suggestion
The following procedure shows you how to check if your cluster has the proper roles and permissions for anonymous pulls and, if it does not, how to enable anonymous pulls.
```